src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        String randomColor = new ColorSupplier().getRandomColor();
+        int randomFigure = random.nextInt(5);
```

Comment:
Think about [magic numbers](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#all-magic-numbers-in-your-code-should-be-constants) in all your code.